### ISSUE TITLE
When saving an event with a Neo field we receive a type error

### ISSUE THREAD

**Issue by a-am** [solspace/craft3-calendar#354]
When saving an event with a Neo field _(with or without content in Neo field)_ we receive a type error.  
```php
Cannot assign string to property benf\neo\Field::$propagationMethod of type craft\enums\PropagationMethod
```
**Location:**
`solspace/craft-calendar/packages/plugin/src/Elements/Event.php` - line 1572.

**Steps to reproduce**

1. Add Neo field to a calendar
2. Create new event _(With or without content in Neo field to trigger error)_
3. Save event

**Expected behavior**
After creating and saving an event the control panel should not receive a type error and be either redirected to the event list or back to the event edit page depending how the event was saved.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 5.5.5 Pro
- Calendar Version: 5.0.12
- Calendar Edition: Pro
- Fresh Install or Upgrade: Fresh Install

**Comment by kjmartens** [comment]
Sorry for the trouble @a-am,

We will investigate and resolve this issue shortly. 🙂

**Comment by kjmartens** [comment]
This should be resolved in Calendar 5.0.13+ now. 🙂 

Thanks for your patience.
